lib/lhm.rb

```diff
@@ -59,9 +59,11 @@ def change_table(table_name, options = {}, &block)
   def cleanup(run = false, options = {})
     lhm_tables = connection.select_values('show tables').select { |name| name =~ /^lhm(a|n)_/ }
     if options[:until]
-      lhm_tables.select! do |table|
```

Comment:
Instead of changing this code block, what about just changing the regex on line 60 to `^lhma_`?